.gitignore

```diff
@@ -20,6 +20,7 @@ latex/
 *.pdf
 *.tag
 .DS_Store
+locm3.sublime-workspace
```

Comment:
oh, it is too!  I hadn't noticed someone had added that.  If it's actually part of the project though, it shouldn't be gitignored.  It should either be dropped, and added to gitignore, or you need to coordinate your changes with other people.